pos_payment_terminal/models/__init__.py

```diff
@@ -0,0 +1,3 @@
+from . import pos_payment_method
+
+# from . import pos_order
```

Comment:
Is it required or deprecated?
if it's deprecated we should remove the file entirely

Reply:
I don't know. I didn't write that code; it was written by Acsone. I don't what is the purpose and how it works. That's why I just commented it and I didn't want to remove it.

Reply:
IIRC this was done to save the payment transaction id in the database when the terminal driver returns it. It [looks like there is native support for that in the point_of_sale](https://github.com/odoo/odoo/blob/66b468626e046a13a463cff3a2285801016f3ed7/addons/point_of_sale/static/src/js/payment.js#L48-L51) module so we should probably use that.

Reply:
I removed this code in https://github.com/akretion/pos/pull/20